common/client/multi_node.go

```diff
@@ -661,21 +662,25 @@ func (c *multiNode[CHAIN_ID, SEQ, ADDR, BLOCK_HASH, TX, TX_HASH, EVENT, EVENT_OP
 func aggregateTxResults(resultsByCode sendTxErrors) (txResult error, err error) {
 	severeErrors, hasSevereErrors := findFirstIn(resultsByCode, sendTxSevereErrors)
 	successResults, hasSuccess := findFirstIn(resultsByCode, sendTxSuccessfulCodes)
+	var severeErrorsList error
+	for _, severeError := range severeErrors {
+		severeErrorsList = multierr.Append(severeErrorsList, severeError)
```

Comment:
`severeErrors` only contains errors of single sever sub type (only Fatal or only Underpriced, etc). 
Aren't we want to collect all the errors?
Also we already log all the errors [here](https://github.com/smartcontractkit/chainlink/blob/f72c899080b524bce66585205882b414b024bad1/common/client/multi_node.go#L655)